src/main/resources/assets/computercraft/lua/rom/programs/move.lua

```diff
@@ -8,11 +8,31 @@ end
 local sSource = shell.resolve(tArgs[1])
 local sDest = shell.resolve(tArgs[2])
 local tFiles = fs.find(sSource)
+
+local function sanityChecks(sSource, sDestination)
+    if fs.isMountPoint(sSource) then
+        printError("Can't move mounts")
+        return false
+    elseif fs.isReadOnly(sSource) then
+        printError("Source is read-only")
+        return false
+    elseif fs.exists(sDest) then
```

Comment:
I feel the destination checks should be done outside the loop - otherwise we'll print them for every file which feels a little odd.